src/py/aspen/config/config.py

```diff
@@ -111,3 +111,7 @@ def URI(self):
     @property
     def READONLY_URI(self):
         raise NotImplementedError()
+
+    @property
+    def INTERFACE(self):
+        raise NotImplementedError()
```

Comment:
What's the rationale behind making the Config object produce the `SqlAlchemyInterface` object?

Reply:
i thought this was a cleaner way to have an interface rather than declaring a global from inside app.py. Especially because the interface depends on attributes already declared within the config. I thought it was also nice with testing that you could create the application from the config directly without also creating an interface fixture object that would have to be shared across fixtures. I'm open to changing it though.

Reply:
I have a preference for keeping the config object to just config things.  I don't mind a new global.  We may have to do more work in terms of using scoped sessions, so I filed a ticket for that.

Reply:
i think i'm already using a scoped session in the usergroup view though

Reply:
oh so also i think that creating a global would be problematic for testing because we're not creating the application from the testing config in app.py, so if we initialize the interface globally in app.py then we would need different logic for also first instantiating the application from the TestConfig (which is problematic becuase the DB_URI isn't stable in the testing environment).

Reply:
Go ahead and merge this as is then.  We can think of a better way to handle this in a future PR.

Reply:
yayyy!!